README.md

```diff
@@ -17,27 +17,20 @@ Building
 The software is written in Java and using Java Server Faces web technology to run on a [Tomcat Servlet container](http://tomcat.apache.org/) backed up by a [MySQL](http://www.mysql.com) database accessed utilizing the [Hibernate framework](http://www.hibernate.org). It uses (Unix/Windows) shell scripts and is often used with Windows shares in [Samba](http://www.samba.org/) environments authenticated via LDAP.
 
 The project structure is IDE independent. The tool for issue building, testing
-and packaging of the application is [Ant](http://ant.apache.org/). Ant gets
-configured by providing a build.properties file next to the build.xml build-file.
+and packaging of the application is [Maven](https://maven.apache.org/).
 
 ### Dependencies
 
-All dependent libraries are shipped with the source code. They can be found in the directory `/Goobi/WEB-INF/lib/`.
+Available dependencies are fetched from Maven Central. Further dependencies(not available in Maven Central) are located in Kitodo\src\main\webapp\WEB-INF\lib
```

Comment:
Please don't use Windows path separators. Use `/` instead.

There is a blank missing before `(`. Also missing is a terminating `.`.